src/main/kotlin/no/nav/familie/ef/sak/brev/dto/FrittståendeBrevDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.brev.dto
+
+data class FrittståendeBrevDto(val overskrift: String,
+                               val avsnitt: List<FrittståendeBrevAvsnitt>,
+                               val fagsakId: String)
```

Comment:
Her er det mulig å bruke `val fagsakId: UUID`, då trenger du ikke å bruke `UUID.fromString(fagsakId)` på det andre stedet denne brukes

Reply:
Vi får denne feilen ved å gjøre om fra String til UUID: `Failed to convert value of type 'java.lang.String' to required type 'java.util.UUID'` 

Er det noe mer vi må gjøre @blommish ?

Reply:
Fikset i a422031